### DISCUSSION TITLE
Moving RouteServiceProvider::HOME to a config / .env value

### DISCUSSION THREAD

**Discussion by Lawebso** [laravel/framework#42514] [Ideas]
Is there a specific reason why we use

```
RouteServiceProvider::HOME
```
instead of allowing to set the value in .env and retrieving it from the config?
```
'route_home_path' => env('ROUTE_HOME_PATH', '/home'),
```

It feels kinda stiff especially if you write your own breeze alike package and you want to use a different path. 

I know I can change it by hand in my installation but it would feel a bit more flexible with the config approach !? Especially for isolated package testing.

**Comment by shaedrich** [comment]
At least `\Illuminate\Routing\Redirector::home()` will be deprecated in 10.x via #42600
